app.vue

```diff
@@ -8,10 +7,10 @@ onMounted(() => {
 </script>
 
 <template>
-  <div id="app">
-    <Listing />
+  <div id="app" class="flex justify-center">
+    <div class="grid grid-cols-2 gap-5 max-w-screen-md">
+      <ListingModified />
```

Comment:
We could also name this Detail.vue. I started with a Listing example and switched over to a single product because of the category context problem. But I did not rename the component. 🙈